modules/codegen/src/main/scala/com/twilio/guardrail/generators/Java/AsyncHttpClientClientGenerator.scala

```diff
@@ -140,7 +141,7 @@ object AsyncHttpClientClientGenerator {
 
     if (isList) {
       new ForEachStmt(
-        new VariableDeclarationExpr(param.argType.containedType, "member", FINAL),
+        new VariableDeclarationExpr(param.argType.containedType, "member", finalModifier()),
```

Comment:
I assume this is essentially a getter, so it'd be proper in scala to leave off the `()`